docs/content/docs/install/manual_demo/_index.md

```diff
@@ -39,44 +39,269 @@ See the [observability documentation](../../patterns/observability/_index.md) fo
+- `bookstore` is a server, which responds to HTTP requests. It is also a client making requests to the `bookwarehouse` service.
+- `bookwarehouse` is a server and should respond only to `bookstore`. Both `bookbuyer` and `bookthief` should be blocked.
 
-- `bookstore-v2` application representing version v2 of the `bookstore` application
 
-The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` demo applications will be installed in their respective Kubernetes Namespaces. For OSM to work, each application Pod must contain an Envoy proxy as a sidecar container. OSM can automatically inject an Envoy proxy sidecar into application Pods. Use the `osm` CLI to add Kubernetes Namespaces to monitor and inject proxies into using the `osm namespace add` command. Once the Namespace is added, OSM will monitor the Namespace for new Pods and automatically inject the Envoy proxy as a sidecar into each Pod created in the Namespace.
+We are going to craft SMI policies, which will bring us to this final desired
+state of allowed and blocked traffic between pods:
 
-### Create the Bookstore Application Namespaces
+| from  /   to: | bookbuyer | bookthief | bookstore | bookwarehouse |
+|---------------|-----------|-----------|-----------|---------------|
+| bookbuyer     |     \     |     ❌     |     ✔    |       ❌       |
+| bookthief     |     ❌     |     \     |     ❌     |       ❌       |
+| bookstore     |     ❌     |     ❌     |     \     |       ✔      |
+| bookwarehouse |     ❌     |     ❌     |     ❌     |      \        |
+
+
+To show SMI Traffic Split, we will deploy an additional application:
+
+- `bookstore-v2` - a server, which is a newer version of the existing `bookstore` application
+
+The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` Pods will be in separate Kubernetes Namespaces with
+the same names. Each new Pod in the service mesh will be injected with an Envoy sidecar container.
+
+### Create the Namespaces
 
 ```bash
-for i in bookstore bookbuyer bookthief bookwarehouse; do kubectl create ns $i; done
+kubectl create namespace bookstore
+kubectl create namespace bookbuyer
+kubectl create namespace bookthief
+kubectl create namespace bookwarehouse
 ```
 
-### Onboard the Namespaces to the OSM Mesh and enable sidecar injection on the namespaces
+### Join the new namespaces to the service mesh
 
 ```bash
-osm namespace add bookstore bookbuyer bookthief bookwarehouse
+osm namespace add bookstore
+osm namespace add bookbuyer
+osm namespace add bookthief
+osm namespace add bookwarehouse
 ```
 
-### Create the Kubernetes resources for the bookstore demo applications
+Now each one of the four namespaces is labelled with `openservicemesh.io/monitored-by: osm` and also
+annotated with `openservicemesh.io/sidecar-injection: enabled`. The OSM Controller, noticing the label and annotation
+on these namespaces, will start injecting all **new** pods with Envoy sidecars.
+
+### Create the Pods and Services
```

Comment:
what about the service accounts?

Reply:
Added ServiceAccounts to the title!
Thank you!